lib/utils/environment.ts

```diff
@@ -75,6 +78,7 @@ export function BuildCommand(command: string, args?: string[], options?: Command
  */
 export function ExecuteCommand(command: Command): Promise<{}> {
 	let tries = ((command.options || {}).retries || 0) + 1;
+	let delay = (command.options || {}).delay || 1000;
```

Comment:
I've seen this pattern in quite a few places in the codebase, it might be better to (eventually) migrate to [_.get()](https://lodash.com/docs/4.17.4#get), which would make these two cases:
```typescript
let tries = (_.get(command.options, 'retries') || 0) + 1
let delay = _.get(command.options, 'delay') || 1000
```
It doesn't really add much for these smaller cases, except perhaps slightly more readable, but there is other places I could see it being quite useful.